### ISSUE TITLE
Sign in with the app-specific password you generated

### ISSUE THREAD

**Issue by m17603048828** [fastlane/fastlane#13303]
default_platform(:ios)

this is my Fastfile

platform :ios do
  desc "Push a new release build to the App Store"
  lane :release do
    build_app(workspace: "001---自动打包技术.xcworkspace", scheme: "001---自动打包技术")
    upload_to_app_store{
       	FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD = "rxpc-vbrs-very-wbrq"
	}
  end
end

[15:49:31]: Successfully uploaded screenshots to App Store Connect
[15:49:33]: Uploading binary to App Store Connect
[15:49:33]: Going to upload updated app to App Store Connect
[15:49:33]: This might take a few minutes. Please don't interrupt the script.
[15:49:39]: [Transporter Error Output]: Sign in with the app-specific password you generated. If you forgot the app-specific password or need to create a new one, go to appleid.apple.com (-22938)
[15:49:39]: Transporter transfer failed.
[15:49:39]: 
[15:49:39]: Sign in with the app-specific password you generated. If you forgot the app-specific password or need to create a new one, go to appleid.apple.com (-22938)

**Comment by m17603048828** [comment]
@janpio 
Yes, I saw your post! But I don't know how to set the app-specific password

**Comment by janpio** [comment]
This should be fixed now in 2.104.0 🚀 Please update and let us know if this fixed it for your. Thanks.
